web/src/providers/Transaction/Transaction.provider.tsx

```diff
@@ -0,0 +1,37 @@
+import {createContext, useContext, useMemo} from 'react';
+import {useGetTransactionByIdQuery} from 'redux/apis/TraceTest.api';
+
+interface IContext {
+  transaction?: ITransaction;
+  version: number;
+}
+
+export interface ITransaction {
+  id: string;
+  name: string;
+  version: number;
+  tests: {name: string; version: number}[];
+  variables: {key: string; value: string; from: string}[];
```

Comment:
Let's move this transaction interface to a centralized types file

Reply:
✅